requirements-doc.txt

```diff
@@ -0,0 +1,7 @@
+
+Sphinx==1.8.5
+sphinx-rtd-theme==0.4.3
+sphinx-markdown-tables==0.0.9
+recommonmark==0.5.0
+doc8==0.8.0
+sphinxcontrib-httpdomain==1.7.0
```

Comment:
Should these really be pinned?

Reply:
Ping

Reply:
Yes, I prefer to fix the sphinx and other libraries versions.. They are doing some work with Sphinx 3.0.0, so I prefer to pin the documentation libraries

Reply:
Why not use `<3.0.0` then? Accept patches, but reject major updates